### DISCUSSION TITLE
[Feature] Clickup Integration

### DISCUSSION THREAD

**Discussion by max-pod** [bergercookie/syncall#124] [Ideas]
Hi, I'm interested in getting a clickup integration and willing to open up a PR.

I saw the thread for the Asana integration,  https://github.com/bergercookie/taskwarrior-syncall/pull/52, but haven't looked further.

Questions:
1. Would a clickup integration PR be accepted?
2. Is there any other pr / commit I can look to understand what I would need provide?
3. Can I / Who should I ask for help and where?

**Comment by bergercookie** [comment]
Hi @max-pod,

Thanks for showing interest!

> Would a clickup integration PR be accepted?

Yes, a clickup integration would be highly appreciated!

> Is there any other pr / commit I can look to understand what I would need provide?

Probably the asana one is the most representative.
There's also this guide on how to create new integrations which could come in handy. Most of the items there should still be relevant: https://github.com/bergercookie/taskwarrior-syncall/blob/master/implement-a-new-synchronization.md

> Can I / Who should I ask for help and where?

Me :-)

**Comment by bergercookie** [comment]
Feel free to ask questions and discuss stuff in the following discussion thread: https://github.com/bergercookie/taskwarrior-syncall/discussions/62

**Comment by bergercookie** [comment]
I 'll close this for now, there doesn't seem to be any active work towards a ClickUp integration and AFAICT there's also no ready python SDK to use with ClickUp.

In any case and for future readers of this issue, any ClickUp related work and PR is more than welcome

**Comment by bergercookie** [comment]
Moving this to a discussion
